ClassLibrary/ClassLibrary/Library.cs

```diff
@@ -1,13 +1,51 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace ClassLibrary
 {
     public class Library
     {
+        //List All Books
         public List<Book> Books= new List<Book>();
-        public List<CardReader> CardReaders = new List<CardReader>();
-        public List<Reader> Readers = new List<Reader>();
+
+        //List All Athor
+        public List<Author> Authors = new List<Author>();
+        //public List<CardReader> CardReaders = new List<CardReader>();
+        //public List<Reader> Readers = new List<Reader>();
+
+        //букк пока не смотри
+        public void AddBook(String ISBN, String NameBook,String FullNameAuthor)
+        {
+            Book newBook = new Book();
+        }
+
+        public Author AddAuthor(String FullNameAuthor)
```

Comment:
Let's have only one way of adding an author (from full name or from first/last names - your choice)

Reply:
And cover the remaining method with unit tests.